Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: 'custom3.2.2-2'
-gem 'omnibus-software', git: 'git://github.com/datadog/omnibus-software.git', branch: 'master'
+gem 'omnibus', :git => 'git://github.com/datadog/omnibus-ruby.git', :branch => 'etienne/ship-licenses-and-sources'
+gem 'omnibus-software', :git => 'git://github.com/datadog/omnibus-software.git', :branch => 'etienne/omnibus-4-migration' 
```

Comment:
Are we building on something nonRuby 1.9 and above? I understand the repo changes, but the hashrocket style should be replaced with the previously existing `git: 'git...`.
Was something not working with the newer style?